tests/unit/conftest.py

```diff
@@ -28,6 +28,13 @@
 from gordon import interfaces
 
 
+PLUGIN_NAMES = ['event_consumer', 'enricher', 'publisher', 'runnable']
```

Comment:
where does `runnable` plugin come from?

Reply:
The context is explained in the PR message itself (LMK if it doesn't make sense). This name refers to the [ability to run plugins that aren't necessarily interface implementations](https://github.com/spotify/gordon/pull/17/commits/2733fc4e7bcdb198154344fd2c23b8418b4d6c61#diff-6ef2c41f2a81ebf1e2f3f7c8065b9160R130).

Does that help?